Enterprise-CMCS/macpro-mdct-mcr
# MDCT-2213: Add "Enter MLR" Button to Home Page

## Description

Closes [MDCT-2213](https://qmacbis.atlassian.net/browse/MDCT-2213).

Adding a new button `Enter MLR` to the MCR homepage and hiding the "available" verbiage behind a feature flag until we're ready to release it.

### How to test

Login as a state user to the deployed instance [here](https://d15iwqrff5qhla.cloudfront.net/).

Check that the `Enter MLR` button is visible on screen! Then ask me to toggle the flag in `LaunchDarkly` for a disappearing act 🪄 

### Changed Dependencies

N/A

## Code author checklist

## Reviewer checklist (two different people)
